api/agent/agent.go

```diff
@@ -127,13 +131,25 @@ func New(da DataAccess) Agent {
 		ServerVersion: "17.06.0",
 	})
 
+	freezeIdleMsecs, err := getEnvMsecs("FN_FREEZE_IDLE_MSECS", 50*time.Millisecond)
+	if err != nil {
+		logrus.WithError(err).Fatal("error initializing freeze idle delay")
+	}
+
+	ejectIdleMsecs, err := getEnvMsecs("FN_EJECT_IDLE_MSECS", 1000*time.Millisecond)
+	if err != nil {
+		logrus.WithError(err).Fatal("error initializing eject idle delay")
+	}
```

Comment:
looked and seems like we're not plumbing agent config stuff very well, no need here, but made a note to allow this to be plumbed from the *Server configuration more easily (seems like a strange issue to file)

Reply:
(there's other things we'll need to / already need to plumb in there)